File: languages/java/testData/variable/notVariableDeclaration.java
package testData.variable;

public class MyClas<caret>s {
    public int myField = 0;
    public static int collision = 1;

    public void setF<caret>ield(int boi) {
        myFi<caret>eld = bo<caret>i;
    }

    public stat<caret>ic void main(String[] args) {
        MyClass instance = new MyClass();
        insta<caret>nce.myField = 1;
    <caret>}
}

File: languages/kotlin/testData/variable/notVariableDeclaration.kt
package variable

class MyClass3(not_collision: Boolean, val collision1: Int = 0) {
    var myField = 0
    private val collision2 = 1
    fun setField(boi: Int) {
        myFi<caret>eld = b<caret>oi
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val instance = MyClass3(true)
            in<caret>stance.my<caret>Field = 1
        }
    }
}